decidim-proposals/spec/queries/decidim/proposals/metrics/proposal_participants_metric_measure_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Proposals::Metrics::ProposalParticipantsMetricMeasure do
+  let(:day) { Time.zone.today - 1.day }
+  let(:organization) { create(:organization) }
+  let(:not_valid_resource) { create(:dummy_resource) }
+  let(:participatory_space) { create(:participatory_process, :with_steps, organization: organization) }
+
+  # Create a proposal (Proposals)
+  # Give support to a proposal (Proposals)
+  # Endorse (Proposals)
```

Comment:
What are these comments for?

Reply:
Reminders for what records are we testing